boa/src/builtins/string/mod.rs

```diff
@@ -230,6 +228,38 @@ impl String {
             .ok_or_else(|| context.construct_type_error("'this' is not a string"))
     }
 
+    /// `String.fromCodePoint(num1[, ...[, numN]])`
+    ///
+    /// The static `String.fromCodePoint()` method returns a string created by using the specified sequence of code points.
+    ///
+    /// More information:
+    ///  - [ECMAScript reference][spec]
+    ///  - [MDN documentation][mdn]
+    ///
+    /// [spec]: https://tc39.es/ecma262/#sec-string.fromcodepoint
+    /// [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/fromCodePoint
+    pub(crate) fn from_code_point(
+        _: &JsValue,
+        args: &[JsValue],
+        context: &mut Context,
+    ) -> JsResult<JsValue> {
+        let mut result = StdString::new();
+
+        for arg in args.iter() {
+            let number = arg.to_number(context)?;
+
+            if !Number::is_float_integer(number) || number < 0.0 || number > f64::from(0x10FFFF) {
+                return Err(
+                    context.construct_range_error(format!("invalid code point: {}", number))
+                );
+            }
+
+            result.push(char::try_from(number as u32).unwrap());
```

Comment:
We shouldn't have an `unwrap()` call (an `expect()` could be fine if we are sure this will never happen).